src/utils/helpers.js

```diff
@@ -135,10 +135,11 @@ const helpers = {
 
   /**
    * Divide a big value to be used with prefixes
-   * The value is divided by 1024 while it can
+   * The value is divided by 1000 while it can, which is the newest convention
+   * for K, M, ... prefixes. Binary prefixes are now Ki, Mi, ...
```

Comment:
Is there a link explaining it?

Reply:
There is this, don't know if it's worth it to add to the docstring.

https://en.wikipedia.org/wiki/Kilobyte

> In December 1998, the IEC addressed such multiple usages and definitions by creating prefixes such as kibi, mebi, gibi, etc., to unambiguously denote powers of 1024.[10] Thus the kibibyte, symbol KiB, represents 2^10  bytes = 1024 bytes. These prefixes are now part of the International System of Quantities. The IEC further specified that the kilobyte should only be used to refer to 1000 bytes.

Reply:
Ok, agreed no link is needed.